code/modules/food_and_drinks/food/foods/bread.dm

```diff
@@ -126,6 +126,25 @@
 	list_reagents = list("nutriment" = 4, "vitamin" = 1)
 	tastes = list("bread" = 10, "cheese" = 10)
 
+/obj/item/reagent_containers/food/snacks/sliceable/banarnarbread
+	name = "banarnarbread loaf"
+	desc = "A loaf of delicious mah'weyh pleggh at e'ntrath!"
+	icon_state = "banarnarbread"
+	slice_path = /obj/item/reagent_containers/food/snacks/banarnarbreadslice
+	slices_num = 5
+	filling_color = "#6F0000"
+	list_reagents = list("nutriment" = 15, "vitamin" = 5)
```

Comment:
Is it intended that eating the loaf whole gives less nutriments than the sum of all slices would?

Reply:
Nope, I'm just bad at maths lol
Fixing now!